app/views/workarea/storefront/searches/autocomplete.html.haml

```diff
@@ -1,31 +1,32 @@
 .grid.grid--rev
-  .grid__cell.grid__cell--25
+  .grid__cell.grid__cell--25-at-medium.grid__cell--100-at-small
     - if @autocomplete.searches.any?
-      %span.search-autocomplete__heading
-        - if @autocomplete.trending_searches?
-          = t('workarea.storefront.search_autocomplete.trending_searches')
-        - else
-          = t('workarea.storefront.search_autocomplete.popular_searches')
+      .search-autocomplete__searches
```

Comment:
I changed the markup here to bring parity to the `__searches` and `__products` elements. They both now follow the same element hierarchy, making them much easier to select via CSS for these responsive changes. This seemed like it would be a smaller diff and be easier to override in themes/host apps rather than jumping through hoops with CSS' cascading in order to get the right things to display. However, it does change the hierarchy of the elements here in `__searches`, so I'm definitely going to need to make some updates to our themes. If you folks are OK with this being a major release, that's probably the best situation since it does include breaking changes.